plutus-playground/configuration.nix

```diff
@@ -0,0 +1,126 @@
+{ config, pkgs, options,... }:
+let 
+  plutusSrc = builtins.fetchGit {
+    url = "https://github.com/input-output-hk/plutus.git";
+    ref = "playground";
+  };
+  plutus = import plutusSrc { inherit config; };
+in
+{
+  imports = [ <nixpkgs/nixos/modules/virtualisation/amazon-image.nix> ];
+  ec2.hvm = true;
+
+  nix = {
+      maxJobs = 8;
+      buildCores = 0;
+      nixPath = options.nix.nixPath.default ++ [
+        "nixpkgs=https://github.com/NixOS/nixpkgs-channels/archive/nixos-18.09.tar.gz"
+      ];
+      binaryCaches = [ https://hydra.iohk.io https://cache.nixos.org https://mantis-hydra.aws.iohkdev.io/ ];
+      requireSignedBinaryCaches = false;
+      extraOptions = ''
+        build-cores = 8
+        auto-optimise-store = true
+      '';
+      trustedBinaryCaches = [ https://hydra.iohk.io https://mantis-hydra.aws.iohkdev.io/ ];
+      binaryCachePublicKeys = [
+        "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY="
+        "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ="
+      ];
+      gc.automatic = true;
+      gc.options = "--delete-older-than 7d";
+    };
+
+  networking.timeServers = [ "1.amazon.pool.ntp.org" "2.amazon.pool.ntp.org" "3.amazon.pool.ntp.org" ];
+
+  networking.firewall = {
+    enable = true;
+    allowedTCPPorts = [ 80 8080 ];
+  };
+
+  users.users.root = {
+    openssh.authorizedKeys.keys = [
```

Comment:
I think it's a bit nicer to use `openssh.authorizedKeys.keyFiles` so you can put them in files and not in big string literals.

Reply:
all gone